pkgs/top-level/all-packages.nix

```diff
@@ -24383,6 +24383,11 @@ in
     cudatoolkit = cudatoolkit_10_2;
   };
 
+  katagoCPU = katago.override {
+    gpuEnabled = false;
+    eigen = eigen;
```

Comment:
This seems useless...

Reply:
The `eigen = eigen` part.  Roger.  I'll take it out.  I know it's a no-op.  I just thought it would make the dependency clearer to a reader.